Documentation/learning/design-auth-v3.md

```diff
@@ -25,7 +25,7 @@ The v3 protocol uses gRPC as its transport instead of a RESTful interface like v
 
 The metadata for auth should also be stored and managed in the storage controlled by etcd's Raft protocol like other data stored in etcd. It is required for not sacrificing availability and consistency of the entire etcd cluster. If reading or writing the metadata (e.g. permission information) needs an agreement of every node (more than quorum), single node failure can stop the entire cluster. Requiring all nodes to agree at once means that checking ordinary read/write requests cannot be completed if any cluster member is down, even if the cluster has an available quorum. This unanimous scheme ultimately degrades cluster availability; quorum based consensus from raft should suffice since agreement follows from consistent ordering.
 
-The authentication mechanism in the etcd v2 protocol has a tricky part because the metadata consistency should work as in the above, but does not: each permission check is processed by the etcd member that receives the client request (etcdserver/api/v2http/client.go), including follower members. Therefore, it's possible the check may be based on stale metadata.
+The authentication mechanism in the etcd v2 protocol has a tricky part because the metadata consistency should work as in the above, but does not: each permission check is processed by the etcd member that receives the client request (server/etcdserver/api/v2http/client.go), including follower members. Therefore, it's possible the check may be based on stale metadata.

```

Comment:
@sid597 thanks for your doc update PRs, this looks good to me but if you are working on doc updates for fixing broken paths (due to recent such changes),  it would be nice to combine the fixes into a single PR as possible. Great job again with fixes! Thank you!

Reply:
@spzala thanks for the reviews, will do so from next commit.

Reply:
Thanks @sid597